README.md

```diff
@@ -3375,6 +3375,7 @@ _Where to discover new Go libraries._
 - [How To Deploy a Go Web Application with Docker](https://semaphoreci.com/community/tutorials/how-to-deploy-a-go-web-application-with-docker) - Learn how to use Docker for Go development and how to build production Docker images.
 - [How to Use Godog for Behavior-driven Development in Go](https://semaphoreci.com/community/tutorials/how-to-use-godog-for-behavior-driven-development-in-go) - Get started with Godog — a Behavior-driven development framework for building and testing Go applications.
 - [Implementing CRUD in Golang REST API with Mux & GORM](https://codewithmukesh.com/blog/implementing-crud-in-golang-rest-api/) - Building a CRUD Golang REST API from scratch using MUX, GORM, MySQL, Viper and clean folder seperation. Entire source code is also included!
```

Comment:
I suggest adding the site in the Websites section instead of each blog post in Tutorials section.

Reply:
I like and agree with the recommendation 🎉

Reply:
@iammukeshm would you mind creating a PR with these changes?

Reply:
#4242